cmd/ocm-backplane/login/login.go

```diff
@@ -283,6 +287,43 @@ func getUsernameFromJWT(token string) string {
 	return claim.(string)
 }
 
+// This function checks the backplane version and aims to only
```

Comment:
Function comments usually start with the function name itself as per the standard convention
```suggestion
// checkBackplaneVersion checks the backplane version and aims to only
```